source/read-write-concern/read-write-concern.rst

```diff
@@ -78,6 +78,15 @@ For naming and deviation guidance, see the `CRUD specification <https://github.c
     level: Optional<ReadConcernLevel | String>
   }
 
+The read concern option is available for the following operations: 
+- ``find`` command
+- ``aggregate`` command and the ``db.collection.aggregate()`` method
```

Comment:
Get rid of the method part. That's shell implementation leaked into the command docs. Also, need to make clear aggregate with $out stage isn't supported.